packages/explat-client/.eslintrc.js

```diff
@@ -0,0 +1 @@
+module.exports = {};
```

Comment:
Feel free to remove that one if we won't be declaring any special `eslint` rules or exceptions here.